Makefile

```diff
@@ -24,6 +24,7 @@ WINDOWS_DISTROS ?=
 DOCKER=DOCKER_CLI_EXPERIMENTAL=enabled docker
```

Comment:
This script also needs to be enhanced to support publishing multiple images:
https://github.com/kubernetes-sigs/sig-storage-local-static-provisioner/blob/master/hack/release.sh#L110

Reply:
It looks like this script would already publish multiple images since it ultimately calls `make cross` https://github.com/kubernetes-sigs/sig-storage-local-static-provisioner/blob/6790dc17afbe0070dac8a01d34d94e1fe102f560/hack/release.sh#L205 and I've added the cleanup-controller image to the `make cross` command in the Makefile. The rest of the release.sh script determines the version number and creates a multi arch manifest for the local provisioner. I would assume the cleanup controller would just use the version of the local provisioner, and I'm wondering if a multi arch manifest is needed for this?

Reply:
The script appears to be doing some checks before "make cross", which may need to be extended to support multiple images:

https://github.com/kubernetes-sigs/sig-storage-local-static-provisioner/blob/master/hack/release.sh#L110

Reply:
Added.